core/src/test/groovy/com/novoda/gradle/release/rule/GradleScriptTemplates.groovy

```diff
@@ -0,0 +1,83 @@
+package com.novoda.gradle.release.rule
+
+import groovy.transform.PackageScope
+
+class GradleScriptTemplates {
+
+    @PackageScope
+    static String java() {
+        return """
+            plugins { 
+                id 'java-library'
+                id 'com.novoda.bintray-release'
+            }
+            
+            repositories {
+                jcenter()
+            }
+            
+            dependencies {
+                implementation "junit:junit:4.12"
+            }
+            
+            publish {
+                userOrg = 'novoda'
+                groupId = 'com.novoda'
+                artifactId = 'test'
+                publishVersion = '1.0'
+            }
+               """
+    }
+
+    @PackageScope
+    static String android() {
+        return """
+            buildscript {
+                repositories {
+                    jcenter()
+                    google()
+                }
+                dependencies {
+                    classpath 'com.android.tools.build:gradle:3.0.0'
+                }
+            }
+            
+            plugins {
+                id 'com.novoda.bintray-release' apply false
```

Comment:
Hmmm, you can as well use `plugins` to use `com.android.library`, can't you? If you can do that, things would be simpler.

Reply:
No we can't do that because the plugin isn't available in the `gradlePluginsPortal()`....
Otherwise we have to adjust the settings.gradle...
Don't want to do this...